src/components/Clock/Clock.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import './Clock.css';
+import moment from 'moment';
+
+class Clock extends React.Component {
+  state = {
+    months: undefined,
+    days: undefined,
+    hours: undefined,
+    minutes: undefined,
+    seconds: undefined
+  };
+
+  componentDidMount() {
+    this.interval = setInterval(() => {
+      const { timeTillDate, timeFormat } = this.props;
```

Comment:
fyi: this.props er de argumentene du har sendt inn. Så det er det du sender inn i app.js der du har lagt inn komponenten